src/test/java/test/suite/UsersTests.java

```diff
@@ -0,0 +1,77 @@
+package test.suite;
+
+import calls.UsersAPI;
+import com.github.javafaker.Faker;
+import data.models.common.EmptyResponse;
+import data.models.users.UsersRequest;
+import data.models.users.UsersResponse;
+import data.provider.UserProvider;
+import jdk.jfr.Description;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import test.asserts.UsersAsserts;
+import test.common.TestBaseUsers;
+
+public class UsersTests extends TestBaseUsers {
+
+    public static String testID;
+    public static String deleteID;
```

Comment:
Ovo nek bude u testu, nema potrebe da izvlacis ovde jer samo u delete testu koristis a i to mozes bez uvodjenja varijable da odradis